### PR TITLE
[eturnal] new guide

### PR DESCRIPTION
**Pull request by thisven**
A promising alternate since coturn seems to be broken on Uberspace (#845).

### CODE REVIEW

File: source/guide_eturnal.rst [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+.. tag_list::
+
+eturnal_ is a modern, straightforward STUN and TURN server written in Erlang. Clients can connect using UDP, TCP, or TLS over IPv4 or IPv6. For authentication, eturnal supports the mechanism described in the `REST API for Access to TURN Services`_ specification.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`firewall ports <basics-ports>
+  * :manual:`cron jobs <daemons-cron>
```
Comment (noave) [inline source/guide_eturnal.rst:28]: links are broken, the last backtick is missing

File: source/guide_eturnal.rst [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+=============
+
+Set up your URL:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Check the `latest release`_ and download a precompiled tarball from https://eturnal.net:
```
Comment (noave) [inline source/guide_eturnal.rst:45]: please explicitly tell to adapt the url accordingly

File: source/guide_eturnal.rst [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+.. code-block:: console
+
+  [isabell@stardust ~]$ openssl dhparam -out $HOME/eturnal/etc/dh-parameters.pem 4096
+  Generating DH parameters, 4096 bit long safe prime, generator 2
+  This is going to take a long time
+  ........+.................................+.....................+..................
+  ....................+.....................................+........+...............
+  [...]
+  [isabell@stardust ~]$ openssl rand -hex 32
+  [isabell@stardust ~]$ <super-long-secret>
```
Comment (noave) [inline source/guide_eturnal.rst:76]: remove the isabell@stardust part in this line

File: source/guide_eturnal.rst [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+  [isabell@stardust ~]$
+
+Backup the standard config: ``mv ~/eturnal/etc/eturnal.yml ~/eturnal/etc/eturnal.yml.bkp``.
+Create a new file at ``~/eturnal/etc/eturnal.yml`` and replace values in brackets ``<value>`` with your values.
+
+.. code-block:: yaml
```
Comment (noave) [inline source/guide_eturnal.rst:82]: the code block is broken, you need to indent it, have a look at https://deploy-preview-1300--uberlab.netlify.app/guide_eturnal/

File: source/guide_eturnal.rst [hunk 5]
```diff
@@ -0,0 +1,242 @@
...
+  [isabell@stardust ~]$ ~/eturnal/bin/eturnalctl daemon
+  [isabell@stardust ~]$ ~/eturnal/bin/eturnalctl info
+  [isabell@stardust ~]$
+
+If it's not running, check your configuration_.
+
+Custom service
+--------------
+
+As eturnal's startup procedure doesn't seem to work as supervisord_ program, it will be controlled using ``~/bin/eturnal`` bash script:
```
Comment (noave) [inline source/guide_eturnal.rst:130]: there is a _ after supervisord

Comment (noave) [inline source/guide_eturnal.rst:130]: please explicitly state to create this file with following content

**Review by noave — CHANGES_REQUESTED** [review]
Thanks, I left some comments and will test the guide after fixing :) I am not sure if we should move the nextcloud and synapse parts to the regarding guides

**Comment by thisven** [comment]
I've managed to add the logo to this PR. I think that's it. Have fun with testing. :smiley:

**Comment by nichtmax** [comment]
Hey @thisven! Just a quick heads up, we haven't forgotten you. Noa is on vacation and will get back to you soon.

**Comment by thisven** [comment]
I updated the guide as eturnal can be managed via supervisorctl if using foreground mode. Thanks to [Thomas](https://johnson.tj) for messaging me about that.

**Comment by luto** [comment]
I added a couple small fixes. Thanks for the guide and sorry about the delay!

**Pull request merged**